### PR TITLE
CSLove page

### LINKED ISSUE #313: The CSLOVE page
Let's add a page to the membership app with the following content:

- Your CSLOVE balance
- Info:
  - What is the CSLOVE and how can you use it? 
  - How do you add it to Metamask?
- Link to store

### PR DESCRIPTION
**Pull request by kristoferlund**
Closes #313

### CODE REVIEW

**Comment by kristoferlund** [comment]
Don't merge, realised I forgot something.

File: src/components/OnboardProvider.js [hunk 1]
```diff
@@ -75,6 +75,22 @@ const Comp = ({ onWeb3Ready, children }) => {
     });
   };
 
+  const addCSLove = async () => {
+    const provider = window.web3.currentProvider;
+    await provider.request({
```
Comment (MerlinEgalite) [inline src/components/OnboardProvider.js:80]: `provider.request()` can be used for `addCSTK` too as `sendAsync` is deprecated now: https://docs.metamask.io/guide/ethereum-provider.html#ethereum-sendasync-deprecated

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]
Just a small change to be up to date with Metamask recommendations

**Review by MerlinEgalite — APPROVED** [review]
LGTM!

**Pull request merged**